wildfly82/README.md

```diff
@@ -0,0 +1,24 @@
+Artificer: Wildfly 8.2
+======================
+
+Artificer is an metadata and software artifact repository comprised of a common data model, multiple interfaces, powerful tools, and exensibility. It implements the [OASIS S-RAMP specification](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=s-ramp) which defines the data model, query language, and an Atom REST binding.
+
+In addition to implementing the spec, Artificer provides a suite of powerful, flexible, and extensible capabilities, described below.
+
+Artificer is 100% open source -- contributions are welcome!
+
+Visit [artificer.jboss.org](http://artificer.jboss.org) to learn more.
+
+## Usage
+
+To start Artificer
+
+    docker run -it artificer/wildfly82[:tag]
```

Comment:
I tend to think that for some products make sense, and for some others just the underlying version is not so important.
It is usually more relevant in a non-docker world, where the application is layered on top of maybe multiple versions during time, and maybe there can be support (JIRAs) opened against the full combination, so it can be relevant.
In a docker world, where you really care about the final project (artificer in this case) people will care more on the specific version of artificer than which underlying wildfly version is ussing. If you wanted to provide many different images, then it makes sense to distinguish, like to distinguish wildfly8 of the upcoming wildfly9. Somebody might want to use a certain version of the underlying container (specially if there are changes in APIs or functionality) in order to provide an heterogeneous deployment.
But IMHO just naming it wildfly would suffice.

And for your second comment, extending the images provides 2 typical use cases:
- Enable admin access to the console (maybe add also debugging capabilities to the container, port 8787 and -debug flag)
- Deploy certain functionality on the image.

Think that images are usually static views, so if I want to have an artificer with my onthologies preloaded, user, roles, schemas, plugins, extensions,... I need to extend the image with my customizations to create my customized image.

Think that the intention of jboss images are always meant to be extended, and used in FROM clauses in Dockerfiles.

Hope it makes sense.